pypsa/plot.py

```diff
@@ -227,11 +224,13 @@ def as_branch_series(ser):
             from shapely.wkt import loads
             from shapely.geometry import LineString
             linestrings = c.df.geometry.map(loads)
-            assert all(isinstance(ls, LineString) for ls in linestrings), \
-                "The WKT-encoded geometry in the 'geometry' column must be composed of LineStrings"
+            assert all(isinstance(ls, LineString) for ls in linestrings), (
+                "The WKT-encoded geometry in the 'geometry' column must be "
+                "composed of LineStrings")
             segments = np.asarray(list(linestrings.map(np.asarray)))
-            if basemap and basemap_present:
-                segments = np.transpose(bmap(*np.transpose(segments, (2, 0, 1))), (1, 2, 0))
+            if drawmap and basemap_present:
```

Comment:
If we show all coastline and so on in the coordinates of `net.srid`, we do not need to reproject the segments. Otherwise you would have to do the equivalent thing with cartopy.

Reply:
this was also just taken from the normal master branch code

Reply:
but broken :)